edx_repo_tools/auth.py

```diff
@@ -141,6 +143,10 @@ def explode(hub, dry):
             hub.organization('edx').iter_repos()
     """
 
+    if getattr(f, '__pass_github_applied', False):
```

Comment:
A comment explaining this would be cool. Also, double-underscore outside of a class is unusual.